docs/docs/platforms/openshift/OPENSHIFT.md

```diff
@@ -0,0 +1,100 @@
+
+=== "Restricted SCC (:octicons-thumbsup-16: Recommended)"
+    !!! info "Restricted SCC"        
+        Restricted SCC denies access to all host features and requires
+        pods to be run with a UID, and SELinux context that are allocated to the namespace.
+    
+    Running containers in OpenShift with a restricted SCC is the most secure approach. To be able to successfully deploy Atlassian DC Helm charts with a restricted SCC, set `openshift.runWithRestrictedSCC` to true in Helm values file:
+
+    ```yaml
+    openshift:
+      runWithRestrictedSCC: true
+    ```
+
+    This property will:
+    
+    * unset `securityContext` on the pod and `initContainers` level, letting OpenShift set it
+    * trigger creation of a ConfigMap with `server.xml` and `seraph-config.xml` that are mounted as read-only files into containers (rather than generated in the container entrypoint). Both `server.xml` and `seraph-config.xml` can be configured in `<product>.tomcatConfig` Helm stanza (except for Bitbucket which does not have these configuration files)
+    * disable nfs-permission fixer init container that is run as root by default. If you need to change permissions in an existing shared-home volume, do it when provisioning/migrating the volume. See [FAQs](../OPENSHIFT_FAQ/#nfs-permission-fixer-is-disabled-how-can-i-make-shared-home-writable) for more details.
+
+=== "Nonroot SCC"
+    !!! info "Nonroot SCC"
+          Nonroot SCC provides all features of the restricted SCC but allows users to run with **any** non-root UID.
+
+    !!! info "Supported Versions"
+        OpenShift nonroot friendly values are available in versions `1.14.0+` for Jira and Confluence, and `1.18+` for Crowd and Bamboo. If you can't upgrade, consider using 'additionalFiles' to mount `server.xml` and `seraph-config.xml` as ConfigMaps which should be created outside the Helm chart.
+
+    To be able to run DC containers as users created during image build you need to let the ServiceAccount use the nonroot SCC. Typically, cluster admin privileges are required (replace namespace and helm release name):
+     
+    ```bash
+    oc adm policy add-scc-to-user nonroot \
+           system:serviceaccount:<namespace>:<helm-release-name>
+    ```
+    
+    Use the following Helm values (replace `jira` and `UIDs` depending on the deployed product):
+    
+    ```yaml
+    jira:
+      securityContext:
+        runAsUser: 2001
+        runAsGroup: 2001
+        fsGroup: 2001
+      tomcatConfig:
+        generateByHelm: true
+      seraphConfig:
+        generateByHelm: true
+    volumes:
+      sharedHome:
+        nfsPermissionFixer:
+          enabled: false  
+    ```
+
+    If you enabled `monitoring.exposeJmxMetrics` you need to run the init container as non root:
```

Comment:
1. Add a comma after `monitoring.exposeJmxMetrics`.
2. Change "non root" to "non-root".